src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -1437,12 +1450,12 @@ static void dump_global_settings(void)
     fprintf(stdout, "max_buffered_send: %" PRIu64 "\n", MPIDI_OFI_global.max_buffered_write);
     fprintf(stdout, "max_buffered_write: %" PRIu64 "\n", MPIDI_OFI_global.max_buffered_send);
     fprintf(stdout, "max_msg_size: %" PRIu64 "\n", MPIDI_OFI_global.max_msg_size);
-    fprintf(stdout, "max_order_raw: %zu\n", MPIDI_OFI_global.max_order_raw);
-    fprintf(stdout, "max_order_war: %zu\n", MPIDI_OFI_global.max_order_war);
-    fprintf(stdout, "max_order_waw: %zu\n", MPIDI_OFI_global.max_order_waw);
-    fprintf(stdout, "tx_iov_limit: %zu\n", MPIDI_OFI_global.tx_iov_limit);
-    fprintf(stdout, "rx_iov_limit: %zu\n", MPIDI_OFI_global.rx_iov_limit);
-    fprintf(stdout, "rma_iov_limit: %zu\n", MPIDI_OFI_global.rma_iov_limit);
+    fprintf(stdout, "max_order_raw: %ld\n", (long) MPIDI_OFI_global.max_order_raw);
+    fprintf(stdout, "max_order_war: %ld\n", (long) MPIDI_OFI_global.max_order_war);
+    fprintf(stdout, "max_order_waw: %ld\n", (long) MPIDI_OFI_global.max_order_waw);
+    fprintf(stdout, "tx_iov_limit: %ld\n", (long) MPIDI_OFI_global.tx_iov_limit);
+    fprintf(stdout, "rx_iov_limit: %ld\n", (long) MPIDI_OFI_global.rx_iov_limit);
+    fprintf(stdout, "rma_iov_limit: %ld\n", (long) MPIDI_OFI_global.rma_iov_limit);
     fprintf(stdout, "max_mr_key_size: %" PRIu64 "\n", MPIDI_OFI_global.max_mr_key_size);
```

Comment:
Why do we need `(long)` casting for `MPI_Aint`?

Reply:
Replaced with `MPI_AINT_FMT_DEC_SPEC`.